package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "6.3.0",
+  "version": "6.3.1",
```

```suggestion
  "version": "6.4.0",
```
Because it's a new feature it should be a `Major` update